incubator/hnc/internal/reconcilers/anchor.go

```diff
@@ -77,7 +77,7 @@ func (r *AnchorReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 	// finalizers.
 	// TODO refactor/split the EX map for 1) reconciler exclusion and 2) subnamespaces exclusion
 	// purposes. See issue: https://github.com/kubernetes-sigs/multi-tenancy/issues/495
```

Comment:
Update?

Reply:
This one was rotten and you mentioned it's not important to fix, so I left it there in case we want to revisit it in the future? It's about splitting the excluded namespaces for reconcilers or self-serving specific in the future if users want different list of namespaces. Deleting this comment will lose the anchor of the issue in the code, WDYT?

Reply:
Ok, maybe just update the name in that case ("EX map")